lib/travis/build/script/elm.rb

```diff
@@ -124,8 +124,16 @@ def elm_format_version
             (config[:elm_format] || elm_version_tagged).to_s
           end
 
+          def version_prefix
```

Comment:
IMO, this method somehow obscures what we want to achieve. Perhaps it's better to roll it in `#elm_version_tagged`:

```ruby
def elm_version_tagged
  if elm_version =~ /^(elm|latest-)/
    elm_version
  else
    "latest-#{elm_version}"
  end
end
```

Reply:
Good point, will change when I get to my computer next :+1:

Reply:
Fixed!